client/src/components/MyLibrary/index.js

```diff
@@ -5,6 +5,7 @@ import ls from 'local-storage';
 import GamesDisplay from "../../components/GamesDisplay";
 import { AppContext } from "../../appContext";
 import "./style.css";
+import { lstat } from "fs";
```

Comment:
what's this?

Reply:
not sure what this is - i may have had a typo on a line and React auto imported what they thought I needed. I'll remove and recommit.